config/locales/en.yml

```diff
@@ -438,6 +441,9 @@ en:
       receive_csv_subsection:
         receive_csv: Get completed forms as CSV files
         title: Optional task
+      translations_section:
+        add_welsh: Edit Welsh language version of your form
```

Comment:
In the designs, this is "Add Welsh language version of your form", I guess one of the them needs updating?